sensing/autoware_pointcloud_preprocessor/schema/approximate_downsample_filter_node.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Approximate Downsample Filter Node",
+  "type": "object",
+  "definitions": {
+    "approximate_downsample_filter": {
+      "type": "object",
+      "properties": {
+        "voxel_size_x": {
+          "type": "number",
+          "description": "the voxel size along x-axis [m]",
+          "default": "0.3"
```

Comment:
Wrote in another PR, but range check is one of the reasons behind schems

Reply:
Thanks! added the boundary in https://github.com/autowarefoundation/autoware.universe/pull/8480/commits/740e4b165863ea32b4e1464a862c28fd3c19bb9a